pkg/term/windows/api.go

```diff
@@ -0,0 +1,343 @@
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+
+	// "github.com/Sirupsen/logrus"
+)
+
+//===========================================================================================================
+// IMPORTANT NOTE:
+//
+//	The methods below make extensive use of the "unsafe" package to obtain the required pointers.
+//	Beginning in Go 1.3, the garbage collector may release local variables (e.g., incoming arguments, stack
+//	variables) the pointers reference *before* the API completes.
```

Comment:
@brendandixon I tried to understand how things are working with the GC, and it's really not clear to me :S Do you, by any chance, have a small reproducible case that would demonstrate the need for `use()` ? Also, from what I've seen, the only time it could be needed to have `use()` is for unsafe pointers that go under the GC's radar, never for values though.